$MirOS: ports/math/ent/patches/patch-ent_html,v 1.1.7.1 2005/03/18 15:50:04 tg Exp $
--- ent.html.orig	Tue Oct 20 17:29:31 1998
+++ ent.html	Sat Jul  7 20:35:35 2007
@@ -123,7 +123,7 @@ The values calculated are as follows:
                 better but still sufficiently non-random to cause
                 concern for demanding applications.  Contrast both of
                 these software generators with the chi-square result
-                of a genuine random sequence created by <a href="/hotbits/">timing
+                of a genuine random sequence created by <a href="http://fourmilab.ch/hotbits/">timing
                 radioactive decay events</a>.
 
 <p>
@@ -136,7 +136,7 @@ The values calculated are as follows:
 <p>
                 See [Knuth, pp.&nbsp;35-40] for more information on the
                 chi-square test.  An interactive
-                <a href="/rpkp/experiments/analysis/chiCalc.html">chi-square
+                <a href="http://fourmilab.ch/rpkp/experiments/analysis/chiCalc.html">chi-square
                 calculator</a> is available at this site.
 
 <p>
@@ -275,6 +275,7 @@ the <em>file_length</em> field in the ty
     algorithm, may achieve greater compression if the file
     contains repeated sequences of multiple bytes.
 
+<!--
 <h2><a href="random.zip"><img src="/images/icons/file.gif" alt="" align=middle width=40 height=40></a> <a href="random.zip">Download random.zip</a> (Zipped archive)</h2>
 
     The program is provided as <a href="random.zip">random.zip</a>,
@@ -285,11 +286,12 @@ the <em>file_length</em> field in the ty
     a 16-bit MS-DOS file which does not require Windows to
     execute), and in source code form along with a
     <code>Makefile</code> to build the program under Unix.
+-->
 
 <h3>SEE ALSO</h3>
 
 <dl compact>
-<dt>&nbsp;    <dd><cite><a href="/rpkp/experiments/statistics.html">Introduction
+<dt>&nbsp;    <dd><cite><a href="http://fourmilab.ch/rpkp/experiments/statistics.html">Introduction
               to Probability and Statistics</a></cite>
 
 <p>
@@ -331,7 +333,7 @@ the <em>file_length</em> field in the ty
 
 <hr>
 <address>
-<a href="/">by John Walker</a><br>
+<a href="http://fourmilab.ch/">by John Walker</a><br>
 October 20th, 1998
 </address>
 </body>
